apps/docs/content/guides/platform/cost-control.mdx

```diff
@@ -0,0 +1,56 @@
+---
+id: 'cost-control'
+title: 'How to control your costs'
+---
+
+## Spend Cap
+The Spend Cap determines whether your organization can exceed the subscription plan's free quota allowance for any metric. Scenarios that could lead to high usage—and thus high costs—include system attacks or bugs in your software. The Spend Cap can protect you from these unexpected costs for certain metrics.
```

Comment:
We might want to clarify two things early on:

- If you are on the free plan, you will never be charged at all
- Spend Cap is only possible on Pro Plan

Reply:
Done